README.md

```diff
@@ -216,6 +217,13 @@ it, simply add the following line to your Podfile:
 pod "Seam3"
 ```
 
+Seam3 is also available through [Carthage](https://github.com/Carthage/Carthage). To install
+it, simply add the following line to your Cartfile:
+
+```
+github "JonnyBeeGod/Seam3"
```

Comment:
Please reference the main Seam3 repo, not your fork

Reply:
Correct, fixed. 

No need for specific carthage examples, see e.g. https://github.com/Alamofire/Alamofire#carthage 

One could add installation instructions for carthage like in the Alamofire repo but honestly I think this is out of scope